domain/consensus/processes/blockprocessor/validateandinsertblockErrorCases_test.go

```diff
@@ -0,0 +1,114 @@
+package blockprocessor_test
```

Comment:
Please call the file `validateandinsertblock_test.go`. This is more like the go convention, and we also don't use camel case for file names